frontend/src/features/public-form/components/FormInstructions/FormInstructionsContainer.tsx

```diff
@@ -8,11 +11,32 @@ export const FormInstructionsContainer = (): JSX.Element | null => {
   const { sectionRefs } = useFormSections()
   const { form, submissionData } = usePublicFormContext()
 
-  return submissionData ? null : (
-    <FormInstructions
-      content={form?.startPage.paragraph}
-      colorTheme={form?.startPage.colorTheme}
-      ref={sectionRefs['instructions']}
-    />
+  const content = useMemo(
+    () => form?.startPage.paragraph,
+    [form?.startPage.paragraph],
+  )
+
+  if (!!submissionData || !content) return null
```

Comment:
/superNit, since the return value of the logic or operator is not used, you need to cast to boolean. It's ok to just assess truthyness:
```suggestion
  if (submissionData || !content) return null
```